drivers/display/CMakeLists.txt

```
@@ -16,6 +16,7 @@ zephyr_library_sources_ifdef(CONFIG_SSD1306		ssd1306.c)
 zephyr_library_sources_ifdef(CONFIG_SSD16XX		ssd16xx.c)
 zephyr_library_sources_ifdef(CONFIG_ST7789V		display_st7789v.c)
 zephyr_library_sources_ifdef(CONFIG_ST7735R		display_st7735r.c)
+zephyr_sources_ifdef(CONFIG_KS0108		display_ks0108.c)
```

please add this to the zephyr library create above.
```suggestion
zephyr_library_sources_ifdef(CONFIG_KS0108 display_ks0108.c)
```